src/node_internals.h

```diff
@@ -311,6 +311,8 @@ void EndStartedProfilers(Environment* env);
 }
 #endif  // HAVE_INSPECTOR
 
+bool HasSignalJSHandler(int signum);
+
```

Comment:
what would be the use case for this for the embedders? Avoid conflicts while installing a handler of its own?

Reply:
ok, reading through the associated commits I got the answer myself. :)